setup.cfg

```diff
@@ -84,7 +84,7 @@ docs =
     pytest
     PyYAML>=3.13
     scipy>=1.1
-    matplotlib>=3.1
+    matplotlib>=3.1, <3.4.0
```

Comment:
When I tried, just pinning it here wasn't enough because `all` overwrites it. I had to pin it like in #11462 .